control/settings.py

```diff
@@ -192,7 +192,31 @@ def abspath(*args):
 CELERY_ALWAYS_EAGER = DEBUG
 
 # Tell Celery where to find the tasks
-CELERY_IMPORTS = ('subsend.tasks','subscription.tasks',)
+CELERY_IMPORTS = (
+    'subsend.tasks',
+    'subscription.tasks',
+    'snappybouncer.tasks',
+)
```

Comment:
@imsickofmaps not sure if this is even needed, the other tasks from servicerating etc... are included by default because Django.